.github/workflows/test.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           cache-name: cache-build
         with:
           path: ./*
```

Comment:
Not sure I understand. What's the practical implication of doing/not doing this?

Reply:
Seems like some tests need `sanity/cli` to be built. If the cache built is not available it will not re-run the build in the `test` step. I don't think it will ever get into that state but it's more off a fail safe if that cached build is not available for one reason or another from the `install` step just fail before running the tests.

Reply:
Ah, yeah, agree. Thanks for clarifying!